src/js/util.js

```diff
@@ -275,43 +275,25 @@ var Util;
         },
 
         getSelectionRange: function (ownerDocument) {
-            var selection = ownerDocument.getSelection();
-            if (selection.rangeCount === 0) {
-                return null;
-            }
-            return selection.getRangeAt(0);
+            this.deprecated('Util.getSelectionRange', 'Selection.getSelectionRange', 'v5.0.0');
+
+            return Selection.getSelectionRange(ownerDocument);
```

Comment:
Well, are you sure? The demo still works great on my Mac and tests seems to be ok with that. Maybe you can give a try on your own to validate there is no such case.

I really prefer to move these functions in Selection.